samples/scripts/PowerShell/GenerateOpenApiYaml.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS 
+Generates the OpenApi doc for the specified version and compares it with the baseline to make sure no breaking changes are introduced
+.Parameter WorkingDir
+The working directory
+.PARAMETER OutputPathofDll
+Path for the web projects dll
+.PARAMETER OutputPathOfOpenApiDoc
+Path to output the OpenApi Doc (in yaml)
+.PARAMETER Versions
+Api versions to generate the OpenApiDoc for and compare with baseline
+#>
+
+param(
+    [string]$WorkingDir,
+
+    [string]$OutputPathofDll,
+
+    [string]$OutputPathOfOpenApiDoc,
+
+    [String[]] $Versions
+)
+foreach($Version in $Versions){
```

Comment:
Can these versions be auto-detected?

Reply:
It doesnt look like they can be with the CLI tool. It requires that a specific version is passed in.